### ISSUE TITLE
Example can only work til 1.4.5

### ISSUE THREAD

**Issue by mallim** [bdemers/shiro-spring-boot-example#1]
Thanks for the great example

Upon changing to

````
<spring-boot.version>1.4.6.RELEASE</spring-boot.version>
````

the following error occurs:

````
Caused by: java.lang.NoSuchMethodError: org.springframework.aop.framework.AopProxyUtils.getSingletonTarget(Ljava/lang/Object;)Ljava/lang/Object;
        at org.springframework.context.event.AbstractApplicationEventMulticaster.addApplicationListener(AbstractApplicationEventMulticaster.java:105)
````

**Comment by darran85** [comment]
any plan to fix this issue?

**Comment by bdemers** [comment]
Fixed in : 20752b3
